api.go

```diff
@@ -31,6 +32,25 @@ import (
 	"github.com/prometheus/alertmanager/version"
 )
 
+var (
+	numReceivedAlerts = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "alertmanager",
+		Name:      "received_alerts_total",
```

Comment:
I'd do alerts_received rather than received alerts, so the link between the two metrics is more obvious.